source/addonHandler/__init__.py

```diff
@@ -175,7 +175,29 @@ def initialize():
 
 def terminate():
 	""" Terminates the add-ons subsystem. """
-	pass
+	emptyTempDocDir()
+
+
+def getTempDocDirPath():
+	return os.path.join(tempfile.gettempdir(), "nvdaDoc")
+
+
+def getTempDocDir():
+	path = getTempDocDirPath()
+	if not os.path.isdir(path):
+		os.mkdir(path)
+	return path

```

Comment:
```suggestion


```
I think this can be removed, or is there any reason to return the path again, even when there's a function to get it?
For example, in config, initConfigPath, the path is created but not returned.

Reply:
Yes, true for current version of the code.

But actually, I am thinking on how to rewrite this code in case the temp "nvdaDoc" directory creation fails for another reason than "already exists" and have not a clear idea on what should be done in this case. Ideas welcome.